style.css

```diff
@@ -0,0 +1,329 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    color:var(--default-black);
+    margin: 0;
+    padding: 0;
+    height: 100vh;
+}
+
+.shadow-box {
+    box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+}
+
+.flex {
+    display: flex;
+}
+
+.flex-center {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+/* navbar */
+
+
+
+.nav-shopy {
+    justify-content: space-between;
+    font-family: 'Roboto', sans-serif;
+    height: 4.5rem;
+    max-width: 80%;
+    margin: auto;
+}
+
+.brand-name {
+    font-size: 1.3rem;
+    color: var(--primary-color);
+}
+
+.silk-img {
+    max-width: 10rem;
+    height: auto;
+}
```

Comment:
In my opinion, the images can be given an `object-fit: cover;` as doing so would not stretch or squeeze the images if the screen size gets reduced. You could check [this](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit) for further reference.